### PR TITLE
Add GoMePubNew Age files

### PR DESCRIPTION
**Pull request by DoobesURU**
Adds age, fni, py and sdl files for the Ae'gura Guild of Messengers Pub.

### CODE REVIEW

**Comment by Hoikas** [comment]
Great! Before I merge this, could you go through the SDL file and "line up" the text, kind of like in the Gehn [Negilahn SDL file](https://github.com/GehnShard/moul-scripts/blob/master/SDL/Negilahn.sdl)? You'll get much better results if you use spaces instead of tabs... :)

**Comment by DoobesURU** [comment]
Spacing in the SDL file has been fixed!

File: SDL/GoMePubNew.sdl [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    
+    VAR BOOL        BarDoorStat[1]              DEFAULT=1
+    VAR BOOL        islmGZBeamVis[1]            DEFAULT=0   DEFAULTOPTION=VAULT
+    VAR BOOL        SinkStat[1]                 DEFAULT=0
+    VAR BOOL        ChisoBookVis[1]             DEFAULT=0   DEFAULTOPTION=VAULT
+}
+
+STATEDESC GoMePubNew
+{
+    VERSION 8
```
Comment (Deledrius) [inline SDL/GoMePubNew.sdl]: I'd really prefer we start at Version 1 for a new Age being added.  These files are prone to grow well enough without coming in from the start with development cruft. ;)

File: Python/GoMePubNew.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+class GoMePubNew(ptResponder):
+
+    def __init__(self):
+        ptResponder.__init__(self)
+        self.id = -1
+        self.version = 1
+
+    def OnFirstUpdate(self):
+            pass
```
Comment (Deledrius) [inline Python/GoMePubNew.py]: Too much indentation here.

**Pull request merged**